docs/configuration/prefill/kvk.rst

```diff
@@ -52,7 +52,7 @@ Configuration
 
    * **Label**: *Fill in a human readable label*, for example: ``My KvK Zoeken API``
    * **Type**: ORC (Overige)
-   * **API root URL**: *The API URL provided by KvK but typically:* ``https://api.kvk.nl/api/v1/zoeken/``
+   * **API root URL**: *The API URL provided by KvK but typically:* ``https://api.kvk.nl/api/v2/zoeken/``
```

Comment:
I think this is wrong since add the `v2/zoeken` part ourselves in the API client

```suggestion
   * **API root URL**: *The API URL provided by KvK but typically:* ``https://api.kvk.nl/api/``
```

which is quite nice since we don't need a data migration now to handle this change